openstack/compute/v2/extensions/keypairs/requests.go

```diff
@@ -76,14 +80,28 @@ func Create(client *gophercloud.ServiceClient, opts CreateOptsBuilder) (r Create
 
 // Get returns public data about a previously uploaded KeyPair.
 func Get(client *gophercloud.ServiceClient, name string) (r GetResult) {
-	resp, err := client.Get(getURL(client, name), &r.Body, nil)
+	r = GetWithUserID(client, name, "")
```

Comment:
`Get` should not be coupled to the new `GetWithUserID`. They need to be two separate functions that can work unrelated.